STP-3186 (1.2): correcting broken fenced code blocks (#1476)
Cray-HPE/docs-csm: operations/node_management/Reboot_NCNs.md

        **`IMPORTANT:`** If the node does not shut down after 5 minutes, then proceed with the power reset below

        1. To power off the node:
           
           ```bash
           ncn-m001# export USERNAME=root
           ncn-m001# export IPMI_PASSWORD=changeme
           ncn-m001# ipmitool -U $USERNAME -E -H ${hostname}-mgmt -I lanplus power off
           ncn-m001# ipmitool -U $USERNAME -E -H ${hostname}-mgmt -I lanplus power status
           ```

           Ensure the power is reporting as off. This may take 5-10 seconds for this to update. Wait about 30 seconds after receiving the correct power status before issuing the next command.

        2. To power back on the node:
           
           ```bash
           ncn-m001# ipmitool -U $USERNAME -E -H ${hostname}-mgmt -I lanplus power on
           ncn-m001# ipmitool -U $USERNAME -E -H ${hostname}-mgmt -I lanplus power status
           ```

           Ensure the power is reporting as on. This may take 5-10 seconds for this to update.

    4. Watch on the console until the node has successfully booted and the login prompt is reached.

>     5. If desired verify method of boot is expected. If the `/proc/cmdline` begins with `BOOT_IMAGE` then this NCN booted from disk:

Reviewer: ```suggestion
    5. If desired, verify that the method of boot is as expected. If the `/proc/cmdline` begins with `BOOT_IMAGE`, then this NCN booted from disk:
```

>         **`IMPORTANT:`** If the node does not shut down after 5 minutes, then proceed with the power reset below

Reviewer: ```suggestion
        **IMPORTANT:** If the node does not shut down after 5 minutes, then proceed with the power reset below.
```

        1. To power off the node:
           
           ```bash
           ncn-m001# export USERNAME=root
           ncn-m001# export IPMI_PASSWORD=changeme
           ncn-m001# ipmitool -U $USERNAME -E -H ${hostname}-mgmt -I lanplus power off
           ncn-m001# ipmitool -U $USERNAME -E -H ${hostname}-mgmt -I lanplus power status
           ```

           Ensure the power is reporting as off. This may take 5-10 seconds for this to update. Wait about 30 seconds after receiving the correct power status before issuing the next command.

        2. To power back on the node:
           
           ```bash
           ncn-m001# ipmitool -U $USERNAME -E -H ${hostname}-mgmt -I lanplus power on
           ncn-m001# ipmitool -U $USERNAME -E -H ${hostname}-mgmt -I lanplus power status
           ```

           Ensure the power is reporting as on. This may take 5-10 seconds for this to update.

    6. Watch on the console until the node has successfully booted and the login prompt is reached.

>     7. If desired verify method of boot is expected. If the `/proc/cmdline` begins with `BOOT_IMAGE` then this NCN booted from disk:

Reviewer: ```suggestion
    7. If desired, verify that the method of boot is as expected. If the `/proc/cmdline` begins with `BOOT_IMAGE`, then this NCN booted from disk:
```

>        If the configurationStatus is `pending`, wait for the job to finish before continuing. If the configurationStatus is `failed`, this means the failed CFS job configurationStatus should be addressed now for this node. If the configurationStatus is `unconfigured` and the NCN personalization procedure has not been done as part of an install yet, this can be ignored.

Reviewer: ```suggestion
       If the `configurationStatus` is `pending`, then wait for the job to finish before continuing. If the `configurationStatus` is `failed`, then the unsuccessful CFS job should be addressed now for this node. If the `configurationStatus` is `unconfigured` and the NCN personalization procedure has not been done as part of an install yet, then this can be ignored.
```

>        If configurationStatus is `failed`, See [Troubleshoot Ansible Play Failures in CFS Sessions](../configuration_management/Troubleshoot_Ansible_Play_Failures_in_CFS_Sessions.md) for how to analyze the pod logs from `cray-cfs` to determine why the configuration may not have completed.

Reviewer: ```suggestion
       If `configurationStatus` is `failed`, then see [Troubleshoot Ansible Play Failures in CFS Sessions](../configuration_management/Troubleshoot_Ansible_Play_Failures_in_CFS_Sessions.md) for how to analyze the pod logs from `cray-cfs` to determine why the configuration may not have completed successfully.
```

>        If configurationStatus is `failed`, See [Troubleshoot Ansible Play Failures in CFS Sessions](../configuration_management/Troubleshoot_Ansible_Play_Failures_in_CFS_Sessions.md) for how to analyze the pod logs from cray-cfs to determine why the configuration may not have completed.

Reviewer: ```suggestion
       If configurationStatus is `failed`, See [Troubleshoot Ansible Play Failures in CFS Sessions](../configuration_management/Troubleshoot_Ansible_Play_Failures_in_CFS_Sessions.md) for how to analyze the pod logs from `cray-cfs` to determine why the configuration may not have completed.
```
